editor/assets/stylesheets/main.scss

```diff
@@ -1,31 +1,30 @@
-body.toplevel_page_gutenberg,
-body.gutenberg_page_gutenberg-demo {
+body {
 	background: $white;
+}
 
-	#update-nag, .update-nag {
-		display: none;
-	}
+#update-nag, .update-nag {
+	display: none;
+}
 
-	#wpcontent {
-		padding-left: 0;
-	}
+#wpcontent {
+	padding-left: 0;
```

Comment:
Per https://github.com/WordPress/gutenberg/pull/2951#issuecomment-338222113, by moving this style out of nesting, we lost specificity. Based on ordering of stylesheets with `admin-menu.css`, the `padding-left: 10px;` wins.